### ISSUE TITLE
[operator-trivy] Change storage for trivy

### ISSUE THREAD

**Issue by hugik-88** [deckhouse/deckhouse#8731]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Use case. Why is this important?

When deploying applications, the trivy operator scans each container and puts all reports into the ETCD database. If there are really a lot of modules, a large number of reports negatively affects the performance of the ETCD database

### Proposed Solution

Add the ability to select a storage for reports, S3 or a local disk or any other type of storage, just not to write reports directly to the api

**Comment by trublast** [comment]
`OPERATOR_SBOM_GENERATION_ENABLED: false` should be enough, bom reports can be fechched from scanner logs

**Comment by deckhouse-BOaTswain** [comment]
This issue has been automatically put in the triage queue because it has not had
recent activity. The team will reconsider the status of this issue. Thank you
for your contributions.

**Comment by a-schedlovsky** [comment]
Inside actual version of TrivyOperator we have trivy.storageClassEnabled, storageClassName, storageSize for store in PV an external reports by json format.

https://github.com/aquasecurity/trivy-operator/blob/d4e7d3c44a6c367fe1686c38198a88d224938124/docs/getting-started/installation/configuration.md  
| OPERATOR_SBOM_GENERATION_ENABLED | true | The flag to enable sbom generation |
| --- | --- | --- |
| OPERATOR_ALTERNATE_REPORT_STORAGE_ENABLED | false | Control where reports are written. By default this is false, so reports will be written normally as CRDs in ETCD memory. However, if you would rather reports be written to a persistent volume, flip this to true. If done a persistent volume claim will be inluded in your installation and all reports will be written there. See settings for more granularity. |

This parametr also good for adding to Deckhouse CR
OPERATOR_WEBHOOK_BROADCAST_URL

In summary, we need 3 options of customizations in Deckhouse CR
